go.mod

```diff
@@ -41,7 +41,7 @@ require (
 	github.com/redhat-appstudio/image-controller v0.0.0-20231003082540-48893226ba8b
 	github.com/redhat-appstudio/integration-service v0.0.0-20231025084434-b3f521c408d1
 	github.com/redhat-appstudio/jvm-build-service v0.0.0-20240126122210-0e2ee7e2e5b0
-	github.com/redhat-appstudio/release-service v0.0.0-20240126184040-c1b86f4617e1
+	github.com/konflux-ci/release-service v0.0.0-20240126184040-c1b86f4617e1
```

Comment:
I would expect the version changes here

Reply:
Yes, I am trying to find the correct version. Hope it will be available once we merge the release-service PR

Once available it will appear in this pkg search https://pkg.go.dev/search?q=konflux-ci&m=package